### ISSUE TITLE
Item fixing with weapons that epic fight breaks right click functionality with (ex: tetra melee weapons)

### ISSUE THREAD

**Issue by SteamDoge9000** [Yesssssman/epicfightmod#917]
Set a night we discussed about on a previous issue regarding a mixin

Open an issue on it here for convenience sake.

When you use a tetra item that is war forge and right click the right click functions do not work. what should happen When using a war forged tool well right clicking (or holding right click) is that the item charges up. Unfortunately some thing with an epic fight is preventing this from happening; and I think it might have to do with the blocking system for some reason

I'll try to provide additional information if needed

**Comment by Yesssssman** [comment]
Guard skill uses the vanilla mechanism of using items. (like pulling a bow or aiming a trident) These have no problems with the items with no functions, but if an item has a special function when using it, I guess the guard skill prevents that. Well, I'd make another mechanism for a guard, but I don't have a proper way to replace this cause the right-click key is the most convenient.

**Comment by SteamDoge9000** [comment]
Well technically speaking a mother ads in tetra compatibility can do this however in order for that to happen there needs to be a way to block the tools ability to do such

if you could make a hook to allow for a blacklist on being able to guard with an item then epic fight NBT might be able to fix this

Basically this will allow the item under specific traits to steal have functionality yet disable the block for tools that may be warforged

All that's need is some sort of way to prevent the mod from even trying to apply the blocking mechanic onto those items

**Comment by SteamDoge9000** [comment]
I asked them about this issue already and they said only you can possibly fix it though you might be able to have them fix it if you can provide the proper ways to allow for certain items while being within the specifications for Weapons that can block to also have some sort of value that his present causes them to be blacklisted from blocking overall… I don't know

**Comment by SteamDoge9000** [comment]
For all we know it could be as simple as the block Key bind being used is masking over tetras r-click

**Comment by SteamDoge9000** [comment]
Maybe taking a look at what epic NBT does might give you an idea on how you can keep the blocking system the way it is

https://github.com/pitbox46/epicfightnbt

And I probably should've said this from the start: even if you don't have the block scale it seems the issue is still be happening

**Comment by Yesssssman** [comment]
Yeah, but I want to tell you the wither & ender dragon animation is not completed yet. I'd contain the fatal bugs like packet buffer mixins in the update, but I'd work on the mod compatibility issues later.

**Comment by SteamDoge9000** [comment]
So will this be fixed for 1.16.5?
